cis/Issues/official/description.xml

```diff
@@ -0,0 +1,33 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Stichting Centraal Informatie Systeem</en>
+		<nl>Demo Stichting Centraal Informatie Systeem</nl>
+	</Name>
+	<ShortName>
+		<en>Demo CIS</en>
+		<nl>Demo CIS</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>official</CredentialID>
+	<Description>
+		<en>Digitaal Paspoort Verzekeren</en>
+		<nl>Digitaal Paspoort Verzekeren</nl>
+	</Description>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.cis.official.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.cis.official.html</nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="InternationaleSanctieCheck">
+			<Name>
+				<en>Internationale Sanctiecheck</en>
+				<nl>Internationale Sanctiecheck</nl>
+			</Name>
+			<Description>
+				<en>Internationale Sanctiecheck</en>
+				<nl>Internationale Sanctiecheck</nl>
+			</Description>
+		</Attribute>
```

Comment:
I advice you to add some additional fields in the credential that refers to the actual person that belongs to this credential. In IRMA, we currently have no issuers that actively check whether a wallet is re-used to fetch information of another user. This means that in theory it could be possible that someone combines the 'internationale sanctiecheck' attribute of user A with the personal data of user B. If you also include a uniquely identifying attribute in the credential (like fullname or something), then verifiers have a choice to  do a check whether the credentials really belong together.

Reply:
I follow to current requirements for the card. I will bring this issue up in the next demo to the stakeholders. It is up to them to decide if they want to include this attribute in their card. Thank you for the advice and feedback.

Reply:
Okay, for demo purposes the current proposal is fine. If at some point they want to do something in production, then this would be an issue to address.